benches/Cargo.toml

```diff
@@ -1,19 +1,19 @@
 [package]
-name = "wgpu-benchmark"
-version.workspace = true
```

Comment:
It's odd seeing those ordered alphabetically. I think usually people expect to see name, version, then everything else.